test/unit/iam-authenticator.test.js

```diff
@@ -67,7 +67,6 @@ describe('IAM Authenticator', () => {
     // verify the scope param wasn't set
     expect(authenticator.scope).toBeUndefined();
     expect(authenticator.tokenManager.scope).toBeUndefined();
-    done();
   });
```

Comment:
I didn't make anything more than superficial changes to these tests to ensure the refactoring maintains compatibility